<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1382</width>
    <height>740</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>740</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1382</width>
    <height>740</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="gl_class" name="glwidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>9</y>
      <width>711</width>
      <height>711</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(215, 215, 215);
border-width : 1.2px;
border-style:solid;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>291</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>60</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>60</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>400</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderScale">
    <property name="geometry">
     <rect>
      <x>939</x>
      <y>400</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_value_scale">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>400</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>90</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertices:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>106</y>
      <width>58</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>120</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Edges:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vertices">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_edge">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_proj">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>540</y>
      <width>111</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Projection</string>
    </property>
    <widget class="QRadioButton" name="radioButton_parallel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Parallel</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_central">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Central</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>540</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Edge</string>
    </property>
    <widget class="QPushButton" name="pushButton_edge_color">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Change color</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_solid_line">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Solid</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_dash_line">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Dash</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_edge_width">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_width_edge">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>70</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>250</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Rotation</string>
    </property>
    <widget class="QLabel" name="label_value_x">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderX">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>90</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Angle X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_value_z">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_value_y">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderZ">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>90</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Angle Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Angle Y</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderY">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>90</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>250</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Movement</string>
    </property>
    <widget class="QLabel" name="label_value_move_x">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Move Z</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderMove_X">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_value_move_z">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_value_move_y">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderMove_Z">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Move X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Move Y</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderMove_Y">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>1000</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>540</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Vertices</string>
    </property>
    <widget class="QPushButton" name="pushButton_vertice_color">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Change color</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_none">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_circle">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Circle</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_vertice_size">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_size_vertice">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>30</number>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_square">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Square</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_background">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>670</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Change background
color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save_image">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>10</y>
      <width>291</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_gif">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>50</y>
      <width>291</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Record gif</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gl_class</class>
   <extends>QWidget</extends>
   <header location="global">view/gl_class.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
